src/main/java/com/exadel/backendservice/dto/EventWithLabelAndDirectionDto.java

```diff
@@ -0,0 +1,25 @@
+package com.exadel.backendservice.dto;
+
+import lombok.Data;
+
+import java.util.Set;
+
+@Data
+public class EventWithLabelAndDirectionDto {
+
+    private String name;
+
+    private String description;
+
+    private String startDate;
```

Comment:
You could use LocalDate instead of String for date field

Reply:
corrected